README.md

```diff
@@ -72,3 +72,54 @@ have.
 
     cd ~/.homesick/repos/dotfiles
     bundle
+
+### Personal dotfiles
+
+If you want to track personal dotfiles that are gitignored, here is an approach.  (Not final, but see the note about `tmux`.)
+
+Make a private repo (`username/my-dotfiles`) with the private dotfiles you want to track.  It should have a structure like this:
+
+```
+.
+└── home
+    ├── .personal
+    │   ├── bin
+    │   │   ├── foo
+    │   │   └── bar
+    │   ├── gitconfig.d
+    │   │   └── user
+    │   ├── tmux
+    │   │   └── user.conf
+    │   ├── vim
+    │   │   └── spell
+    │   │       ├── en.utf-8.add
+    │   │       └── en.utf-8.add.spl
+    │   └── zsh
+    │       └── local.zsh
+    └── .vimrc.local
+```
+
+Then add another `homesick` castle:
+
+```
+homesick clone username/my-dotfiles
+homesick symlink my-dotfiles --force
```

Comment:
does order matter here? I'm assuming this overwrites any existing symlinks

Reply:
Yup, last step.  Will update.